class/validation.js

```diff
@@ -0,0 +1,205 @@
+
+/**
+ * Function to check only alphabet and spaces in given name field.
+ * 
+ *  @param fieldName
+ *    Field name where to check the error.
+ *  @param errorFieldName
+ *    Name of the field to display error if occured.
+ *    
+ * */ 
+function allLetter(fieldName, errorFieldName) {
+  document.getElementsByName(fieldName)[0].onchange = function () {
+    inputtxt = document.getElementsByName(fieldName)[0].value;
+    var pattern = /^[A-Za-z-' ]+$/;
+    if (inputtxt.match(pattern)) {
+      document.getElementsByName(errorFieldName)[0].innerHTML = "";
+    } 
+    else {
+      document.getElementsByName(errorFieldName)[0].innerHTML =
+        "Only letters and white space allowed";
+    }
+  };
+}
+
+/**
+ * Function to check valid Indian phone no.
+ * 
+ *  @param fieldName
+ *    Field name where to check the error.
+ *  @param errorFieldName
+ *    Name of the field to display error if occured.
+ *    
+ * */
+function validPhone(fieldName, errorFieldName) {
+  document.getElementsByName(fieldName)[0].onkeyup = function () {
+    inputtxt = document.getElementsByName(fieldName)[0].value;
+    var pattern = /^[+][9][1][6-9][0-9]{9}$/;
+    if (inputtxt.match(pattern)) {
+      document.getElementsByName(errorFieldName)[0].innerHTML = "";
+    } 
+    else {
+      if (inputtxt.slice(0, 3) != "+91") {
```

Comment:
why you're nesting if inside an else block.
Simply use an else if block instead.